docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,153 @@
 # Integrating the Benefits API
 
-Once an employer has connected their system to your application, you can use Finch's [Benefits API](https://developer.tryfinch.com/docs/reference/b3A6MTg4Mzc2MTk-create-benefit) to manage benefits (401k, HSA, Loans, etc).
+Finch's [Benefits API](https://developer.tryfinch.com/docs/reference/b3A6MTg4Mzc2MTk-create-benefit) endpoints allow users to read and write benefits information (401k, Health Savings Accounts, Loans, etc) across providers using a single, unified API.
 
-Working with benefits through the Finch API is a simple two-phase process:
-1. Benefits are configured at the company level. When a new benefit is created or an existing benefit is read from the system, you will receive a `benefit_id` for it.
-2. You can use the `benefit_id` to manage the enrollment of employees in that benefit.
+## Calling Finch’s API
 
-----
+Before creating and enrolling employees in benefits, you must create an access token using our Finch Connect flow. If the provider is listed as an [Automated API Provider](https://developer.tryfinch.com/docs/reference/96f5be9e0ec1a-providers#automated-api-providers), you will the [Automated Connect Flow](https://developer.tryfinch.com/docs/reference/a2c944f1041f6-automated-connect-flow). If the provider is listed as an [Assisted API Provider](https://developer.tryfinch.com/docs/reference/96f5be9e0ec1a-providers#assisted-api-providers), you will use our [Assisted Connect Flow](https://developer.tryfinch.com/docs/reference/8c540ddeca222-assisted-connect-flow). Once you have an access token, you can call the Finch API normally.
+
+1. Both **Automated** and **Assisted** Benefits connections can call these endpoints
```

Comment:
I think we should add a section for the /job endpoint that only Assisted can call